.travis/deploy_npm

```diff
@@ -38,16 +37,15 @@ exe npm -v
 # for token authentication we need newest npm!
 exe npm i npm@latest -g
 exe npm -v
+
+# get qooxdoo compiler, must be changed to @qooxdoo/compiler later
+npm install qxcompiler 
+# compile framework
+npx qx compile -v
+
+
 # fill .npmrc with access token
 echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
 # publish to npm
-exe npm publish 
-
-# build qooxdoo server package
-cd $BASE_DIR/component/standalone/server 
```

Comment:
why are we removing this - has it been replaced?

Reply:
Yes. Server package becomes part of @qooxdoo/framework compiled by qx:


# get qooxdoo compiler, must be changed to @qooxdoo/compiler later 
npm install qxcompiler  
# compile framework 
npx qx compile -v